composio/local_tools/local_workspace/config/default.yaml

```diff
@@ -1,96 +1,5 @@
-     Then start trying to fix it.
-     When you think you've fixed the bug, re-run the bug reproduction script to make sure that the bug has indeed been fixed.
-     
-     If the bug reproduction script does not print anything when it successfully runs, we recommend adding a print("Script completed successfully, no errors.") command at the end of the file,
-     so that you can be sure that the script indeed ran fine all the way through. 
-
-  2. If you run a command and it doesn't work, try running a different command. A command that did not work once will not work the second time unless you modify it!
-
-  3. If you open a file and need to get to an area around a specific line that is not in the first 100 lines, say line 583, don't just use the scroll_down command multiple times. Instead, use the goto 583 command. It's much quicker. 
-     
-  4. If the bug reproduction script requires inputting/reading a specific file, such as buggy-input.png, and you'd like to understand how to input that file, conduct a search in the existing repo code, to see whether someone else has already done that. Do this by running the command: find_file "buggy-input.png" If that doesn't work, use the linux 'find' command. 
-
-  5. Always make sure to look at the currently open file and the current working directory (which appears right after the currently open file). The currently open file might be in a different directory than the working directory! Note that some commands, such as 'create', open files, so they might change the current  open file.
-
-  6. When editing files, it is easy to accidentally specify a wrong line number or to write code with incorrect indentation. Always check the code after you issue an edit to make sure that it reflects what you wanted to accomplish. If it didn't, issue another command to fix it.
-     
-
-  (Open file: {open_file})
-  (Current directory: {working_dir})
-  bash-$
-next_step_template: |-
-  {observation}
-  (Open file: {open_file})
-  (Current directory: {working_dir})
-  bash-$
-next_step_no_output_template: |-
-  Your command ran successfully and did not produce any output.
-  (Open file: {open_file})
-  (Current directory: {working_dir})
-  bash-$
-demonstration_template: |
-  Here is a demonstration of how to correctly accomplish this task.
-  It is included to show you how to correctly use the interface.
-  You do not need to follow exactly what is done in the demonstration.
-  --- DEMONSTRATION ---
-  {demonstration}
-  --- END OF DEMONSTRATION ---
-state_command:
-  name: state
-  code: |
-    state() {
-      local working_dir="$PWD";
-      if [ -z $CURRENT_FILE ]; then
-          echo '{"open_file": "n/a", "working_dir": "'$working_dir'"}';
-      else
-          echo '{"open_file": "'$(realpath $CURRENT_FILE)'", "working_dir": "'$working_dir'"}';
-      fi
-    };
-parse_function: ThoughtActionParser
+system_template:
```

Comment:
Just commenting for notice, this is intended ?

Reply:
Ha for some random reason, it used to fail without this. Still a lot of cleanup is pending, will delete this in next one.